### DISCUSSION TITLE
Using CRON and error related to 'expire_at'

### DISCUSSION THREAD

**Discussion by ghost** [deta/cloud-docs#609] [Help]
I can run this code locally and insert data while using the expire_at key word. But when I try to do the same thing with code deployed on DETA and using CRON I get the error: TypeError: insert() got an unexpected keyword argument 'expire_at'

`from deta import Deta
from deta import app
import datetime
import requests

deta = Deta('a0icdh3t_S3fjAUgo41jpRBYQZUa3GQQAzxBksBn4')
news = deta.Base('news')

@app.lib.cron()
def update_datebase(event):

    state = 'alabama'  
    try:
        url = f'http://api.nytimes.com/svc/semantic/v2/concept/name/nytd_geo/{state}?fields=all&api-key=api_key'
        response = requests.get(url)
        text = response.json()
        body = text['results'][0]['article_list']['results'][0]['body']
        url = text['results'][0]['article_list']['results'][0]['url']
        title = text['results'][0]['article_list']['results'][0]['title']
        cat = 0
    except Exception as e:
        title = f'problem with {state}, {e}'
        body = f'problem with {state}, {e}'
        url = f'problem with {state}, {e}'
        cat = f'problem with {state}, {e}'

    expire_time = datetime.datetime.today() + datetime.timedelta(days=7)
    date = datetime.datetime.today().date().isoformat()
    news.insert({'state': state, 'title': title, 'body': body, 'url': url, 'date': date, 'cat':cat}, expire_at=expire_time)

`

**Comment by lemonyte** [comment]
You must list `deta` in your `requirements.txt` file so that the latest version of the SDK is installed, the pre-installed version is out of date.
Also, please delete and regenerate your project key immediately, since you've shared it here.

**Reply by ghost** [reply]
Super! I did what you said and it works. Thanks for the quick response. I also deleted that old api key...
